pkg/cluster/deploybaseresources.go

```diff
@@ -444,6 +454,60 @@ func (m *manager) setMasterSubnetPolicies(ctx context.Context) error {
 	return err
 }
 
+func (m *manager) generateFederatedIdentityCredentials(ctx context.Context) error {
```

Comment:
Nit: I'd call this `federateIdentityCredentials` - we aren't "generating" anything per say; we are "federating" the identities.

Reply:
updated